provider/resources.go

```diff
@@ -288,6 +294,11 @@ func Provider() tfbridge.ProviderInfo {
 					Source: "database_secret_backend_role.md",
 				},
 			},
+			"vault_identity_mfa_duo":                    {Tok: makeResource(identityMod, "MfaDuo")},
+			"vault_identity_mfa_login_enforcement":      {Tok: makeResource(identityMod, "MfaLoginEnforcement")},
+			"vault_identity_mfa_okta":                   {Tok: makeResource(identityMod, "MfaOkta")},
+			"vault_identity_mfa_pingid":                 {Tok: makeResource(identityMod, "MfaPingID")},
```

Comment:
by convention we do capitalize only after the `_` character, so we should use
`"MfaPingid"`